hpx/util/bind.hpp

```diff
@@ -84,27 +82,27 @@ namespace hpx { namespace util
             static HPX_FORCEINLINE
             type call(T& t, Us&& /*unbound*/)
             {
-                return std::move(t);
+                return std::forward<T>(t);
```

Comment:
`T` is a forwarding reference in this context, it will be an lvalue-reference when the bound argument is a `reference_wrapper`.